docs/languages/csharp/properties.md

```diff
@@ -0,0 +1,345 @@
+# Properties
+
+Author: [Bill Wagner](https://github.com/BillWagner)
+
+Properties are first class citizens in C#. The language
+defines syntax that enables developers to write code
+that accurately expresses their design intent.
+
+Properties behave like fields when they are accessed.
+However, unlike fields, properties are implemented
+with accessors that define the statements executed
+when a property is accessed or assigned.
+
+# Property Syntax
+The syntax for properties is a natural extension to
+fields. A field defines a storage location:
+
+```cs
+public class Person
+{
+	public string FirstName;
+	// remaining implementation removed from listing
+}
+```
+
+A property definition contains declarations for a `get` and
+`set` accessor that retrieves and assigns the value of that
+property:
+
+```cs
+public class Person
+{
+	public string FirstName
+	{
+		get;
+		set;
+	}
+	// remaining implementation removed from listing
+}
+```
+
+The syntax shown above is the *auto property* syntax. The compiler
+generates the storage location for the field that backs up the 
+property. The compiler also implements the body of the `get` and `set` accessors.
+You can also define the storage yourself, as shown below:
+
+```cs
+public class Person
+{
+	public string FirstName
+	{
+		get { return firstName; }
+		set { firstName = value; }
+	}
+	private string firstName;
+	// remaining implementation removed from listing
+}
+```
+ 
+The property definition shown above is a read-write property. Notice
+the keyword `value` in the set accessor. The `set` accessor always has
+a single parameter named `value`. The `get` accessor must return a value
+that is convertable to the type of the property (`string` in this example).
```

Comment:
Overall everything LGTM! But one last edit perhaps: should this be convertible instead?